### PR TITLE
Added Terra Bridge 

### PR DESCRIPTION
**Pull request by wrappor**
##### Twitter Link:
https://twitter.com/terra_money

##### List of audit links if any:
None I could find

##### Website Link:
https://bridge.terra.money/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
https://twitter.com/terra_money/photo

##### Chain: Terra 

Coingecko ID (so your TVL can appear on Coingecko): No Token
Coinmarketcap ID (so your TVL can appear on Coinmarketcap): No Token

##### Short Description (to be shown on DefiLlama):
```Terra Bridge enables cross-chain transfer of all tokens supported by Shuttle, including Terra native tokens, most mAssets and also other token types from Terra ecosystem.```

##### methodology (what is being counted as tvl, how is tvl being calculated):
The assets are all stored on terra wallets designated for the bridges (Treasury type TVL). This PR sums together the supported tokens in each wallet to come up with the total.

### CODE REVIEW

**Comment by 0xngmi** [comment]
an issue with hardcoding the list of assets like that is that it'll easily become outdated, any thoughts on how to change this in order to get it to become self-updating?

**Comment by 0xngmi** [comment]
also, we count bridge tvl at the chain where assets have been ported, not origin chain

**Comment by wrappor** [comment]
> an issue with hardcoding the list of assets like that is that it'll easily become outdated, any thoughts on how to change this in order to get it to become self-updating?

yeah - I'll take a look to see if there's a query somewhere to get a better token list for the bridge. I guess we would still need to maintain a list of coingecko mappings for getting prices? (e.g.`'terra14z56l0fp2lsf86zy3hty2z47ezkhnthtr9yq76': 'anchor-protocol'`)

**Comment by wrappor** [comment]
> 

found something we could use here: `https://assets.terra.money/cw20/tokens.json`. again, probably have to map to coingecko ids either way, but at least it's easier this way

**Comment by wrappor** [comment]
updated the data to use something more dynamic and bridged (minted) tokens instead of origin chain.

File: projects/terra-bridge/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        const chainData = data[chain];
+        const block = await getBlock(timestamp, chain, chainBlocks);
+        const chainTokens = await fetchAssets(chainData.path);
+
+        const transform = 'transform' in chainData ? await chainData.transform: false;
+        for (const [tokenAddress, chainAddress] of Object.entries(chainTokens.data.mainnet)) {
+            const bridgedSupply = await sdk.api.erc20.totalSupply({ 
+                block, chain, target: chainAddress
+            });
+            transform ?
```
Comment (0xngmi) [inline projects/terra-bridge/index.js:32]: sdk.util.sumSingleBalance(balances, (transform ?? id=>id)(chainAddress), bridgedSupply.output)
?

**Pull request merged**